Homework3: Ruslan Mazouka

6441066 opened the pull request.

6441066 committed:
Homework3

6441066 committed:
Homework 3

6441066 committed:
Homework 3 Refactoring

6441066 committed:
Homework 3 Refactoring 2

6441066 committed:
Homework 3 Refactoring 3

6441066 committed:
Homework 3 + Chocolate (1 and 2  parts)

6441066 committed:
Homework 3 + Chocolate (1 and 2 parts)

6441066 committed:
Homework3 Refactoring

6441066 committed:
Homework3 Change title of last commit

6441066 committed:
Homework 1 after corrections:

1. Made two lines between functions correctly.
2. Wrote title for all tasks
3. Removed files Homework2
4. Removed unnecessary files.
5. Delete pieces of commented code.
6. Didn't find problems with __init__.py. It is empty.
7. Worked on a code style.
8. There are all tasks in Homework1.py and one task in testing_doctest.py.
9. Separated all functions with two empty lines.
10. PR have name: Homework1 Ruslan Mazouka
11. I didn't touch __init__.py. I can't understand this problem.

6441066 committed:
Merged Homework3

6441066 committed:
Homework 3 refactoring:

6441066 committed:
Homework 3 refactoring 4:

belovmd reviewed (changes requested):
Thank you for your work. 
it looks great
Please address next paragraphs from review notes [Review Notes](https://github.com/belovmd/it-academy-python-spring/blob/master/homework_review/Homework3.md):
- 10
- 11

6441066 committed:
Homework 3 error correction 1:

Tasks with files of results in direcory "data5".
In directory "pack1" - file tasks4.py with tasks for 'runner'.

6441066 committed:
Homework 3 error correction 2:

belovmd reviewed (commented):
9,5
